tools/configure.bat

```diff
@@ -1,158 +1,158 @@
-if "%1"=="-h" goto :ShowUsage
-if "%1"=="-d" goto :SetDebug
-if "%1"=="-f" goto :SetFormat
-if "%1"=="-b" goto :SetFormat
-if "%1"=="-l" goto :SetHostOption
-if "%1"=="-c" goto :SetHostOption
-if "%1"=="-n" goto :SetHostOption
-if "%1"=="-L" goto :SetList
-if "%1"=="-a" goto :SetAppDir
-
-set config=%1
-goto EndOfLoop
-
-:SetDebug
-set debug=%1
-goto :NextArg
-
-:SetFormat
-set fmt=%1
-goto :NextArg
-
-:SetHostOption
-set hostopt=%1
-goto :NextArg
-
-:SetList
-set list=%1
-goto :EndOfLoop
-
-:SetAppDir
-shift
-set appdir=-a %1
-
-:NextArg
-shift
-goto :ArgLoop
-
-:EndOfLoop
-
-rem Check if we have to build configure.exe
-
-if exist configure.exe goto :HaveConfigureExe
-
-set cc=mingw32-gcc.exe
-set cflags=-Wall -Wstrict-prototypes -Wshadow -g -pipe -I. -DCONFIG_WINDOWS_NATIVE=y
-echo %cc% %cflags% -o configure.exe configure.c cfgparser.c
-%cc% %cflags% -o configure.exe configure.c cfgparser.c
-if errorlevel 1 (
-  echo ERROR: %cc% failed
-  echo Is ming32-gcc.exe installed?  Is it in the PATH variable?
-  goto End
-)
-
-:HaveConfigureExe
-cd ..
-tools\configure.exe %debug% %fmt% %hostopt% %appdir% %config% %list%
-if errorlevel 1 echo configure.exe failed
-goto End
-
-:NoConfig
-echo Missing ^<board-name^>:^<config-name^> argument
-
-:ShowUsage
-echo USAGE: %0 [-d] [-b|f] [-a ^<app-dir^>] ^<board-name^>:^<config-name^>
-echo        %0 [-h]
-echo\nWhere:
-echo  -d:
-echo    Enables debug output
-echo  -b:
-echo    Informs the tool that it should use Windows style paths like C:\\Program Files
-echo    instead of POSIX style paths are used like /usr/local/bin.  Windows
-echo    style paths are used by default.
-echo  -f:
-echo    Informs the tool that it should use POSIX style paths like /usr/local/bin.
-echo    By default, Windows style paths like C:\\Program Files are used.
-echo  -l selects the Linux (l) host environment.  The [-c^|n] options
-echo    select one of the Windows environments.  Default:  Use host setup
-echo    in the defconfig file
-echo  [-c^|n] selects the Windows host and a Windows environment:
-echo    Cygwin (c), or Windows native (n). Default Cygwin
-echo  -L:
-echo    List all available configurations.
-echo  -a ^<app-dir^>:
-echo    Informs the configuration tool where the application build
-echo    directory.  This is a relative path from the top-level NuttX
-echo    build directory.  But default, this tool will look in the usual
-echo    places to try to locate the application directory:  ../apps or
-echo    ../apps-xx.yy where xx.yy is the NuttX version number.
-echo  ^<board-name^>:
-echo    Identifies the board.  This must correspond to a board directory
-echo    under nuttx/boards/.
-echo  ^<config-name^>:
-echo    Identifies the specific configuration for the selected ^<board-name^>.
-echo    This must correspond to a sub-directory under the board directory at
-echo    under nuttx/boards/^<board-name^>/configs/.
-echo  -h:
-echo    Prints this message and exits.
-
-:End
+@echo off
+
+rem tools/configure.bat
```

Comment:
@anchao look like the script handle \r\n incorrectly?

Reply:
Yes, I have reverted the incorrect part.